symbols/arm9.yml

```diff
@@ -3144,6 +3144,37 @@ arm9:
         
         r0: animation control
         r1: sprite id in WAN_TABLE
+    - name: SetAnimationForAnimationControlInternal
+      address:
+        EU: 0x201C218
+      description: |-
+        Set the wan animation (and other related settings) of an animation_control
+        Used by SetAnimationForAnimationControl
+        
+        r0: animation_control
+        r1: wan_header
+        r2: animation group id
+        r3: animation id
+        stack[0]: ?
+        stack[1] (0x4): palette pos low (see the field on animation_control)
+        stack[2] (0x8): ?
+        stack[3] (0xC): ?
+        stack[4] (0x10): palette_bank (directly set to the animation_control field with said name)
+    - name: SetAnimationForAnimationControl
+      address:
+        EU: 0x201C368
+      description: |-
+        Set the animation to play with this animation control, but do not start it.
+        
+        (args same as SetAndPlayAnimationForAnimationControl)
+        r0: animation_control
+        r1: animation key (either an animation or animation group depending on the type of sprite and if it does have animation group with this animation key as index)
+        r2: direction_id (unsure) (the key to the wan_animation in itself, only used when animation key represent a wan_animation_group)
+        r3: ?
+        stack[0]: low_palette_pos
+        stack[1] (0x4): ?
+        stack[2] (0x8): ?
+        stack[3] (0xC): ?
```

Comment:
This last parameter isn't given in the function header you defined. It should either be removed here, or added to the function header.